src/codecs/rnb/index.ts

```diff
@@ -138,7 +138,7 @@ export class RnbCodec extends Codec implements Codec {
       const html = container.innerHTML
       container.innerHTML =
         html.slice(0, begin) +
-        `<stencila-codeexpression>
+        `<stencila-codeexpression programming-language="r">
```

Comment:
To be consistent with https://github.com/stencila/designa/pull/17#discussion_r324853763 shall we change to `<stencila-code-expression>` and `<stencila-code-chunk>` as part of this PR?

Reply:
I was planning on changing things over in an upcoming PR, once I expose the `elementName` from the components

Reply:
OK, thanks